### PR TITLE
Fix code format in Single Disk Snapshot feature page

### PR DESCRIPTION
**Pull request by rollandf**
Changes proposed in this pull request:

- Fix code format

- Add missing feature info

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): @rollandf

This pull request needs review by: @mureinik

### CODE REVIEW

File: source/develop/release-management/features/storage/single-disk-snapshot.html.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ authors: amureini, derez, vered
 wiki_title: Features/Single Disk Snapshot
 wiki_revision_count: 14
 wiki_last_updated: 2014-05-25
+feature_name: Single Disk Snapshot
+feature_modules: storage
```
Comment (mureinik) [inline source/develop/release-management/features/storage/single-disk-snapshot.html.md]: AFAIK, "modules" here means "projects", so this should state "engine"

File: source/develop/release-management/features/storage/single-disk-snapshot.html.md [hunk 2]
```diff
@@ -20,16 +24,15 @@ Customization of snapshots with regards to VM configuration and disks.
...
+*   Status: released
 
 ## Benefit to oVirt
 
 *   Allow creation of a customized snapshot. I.e. selecting which disks to take snapshot on.
 *   Custom snapshot preview:
     -   Previewing a new state by selecting VM configuration and disks from various snapshots.
     -   Support commit to the new state and undo to the previous one.
-*   Features based on specific disks snapshots, such as [Storage Live Migration](Features/Design/Storage Live Migration), will benefit from the new support
+*   Features based on specific disks snapshots, such as [Storage Live Migration](/develop/release-management/features/storage/storagelivemigration), will benefit from the new support
```
Comment (mureinik) [inline source/develop/release-management/features/storage/single-disk-snapshot.html.md]: I missed this one when relocating the LSM page. Thanks!

File: source/develop/release-management/features/storage/single-disk-snapshot.html.md [hunk 3]
```diff
@@ -68,41 +71,53 @@ Add <disks> tag to create/restore snapshot.
...
+ <disks>
+  <disk id="{disk_id}">
+   <image_id>"{image_id}"</image_id>
+   <snapshot id="{snapshot_id}"/>
+  </disk>
+ </disks>
+</action>
+```
 
 #### oVirt 3.5
```
Comment (mureinik) [inline source/develop/release-management/features/storage/single-disk-snapshot.html.md]: I don't understand this title.

Comment (rollandf) [inline source/develop/release-management/features/storage/single-disk-snapshot.html.md]: Maybe the "Preview Snapshot" part was added only on 3.5.
In any case it needs rewording.

@danielerez  can you confirm it was added later, in 3.5 ?

**Comment by mureinik** [comment]
@rollandf Can you please rebase this so it can be merged?

**Comment by rollandf** [comment]
Rebased

**Comment by bproffitt** [comment]
Merged #471

**Pull request merged**